go-manual/modules/ROOT/pages/install.adoc

```diff
@@ -0,0 +1,41 @@
+= Installation
+
+To start creating a Neo4j Go application, you first need to install the Go Driver and get a Neo4j database instance to connect to.
+
+[#install-driver]
+== Install the driver
+
+If you are starting from scratch, the first step is to initialize a Go module.
+You can do so by creating a directory, entering it, and using `go mod init`:
+
+[source, bash]
+----
+mkdir neo4j-app
+cd neo4j-app
+go mod init neo4j-app
```

Comment:
That is a practice I can't find a nice way to adhere to, and in fact I have seen many tutorials/examples online that do _not_ use it with a URI, but just with a name as I did. The point is that we want to make it as painless as possible for users to copy-paste and get up and running, and if we put a URI ... what should it be? Asking them to make a choice and the _action_ of creating a repo somewhere is a large friction point. I'd rather them get started with a non-best-practice setup, enjoy using Neo4j, and then spending 5 minutes one week later to fix their setup, rather than feeling that it's complicated to get started.

Happy to hear solution ideas you may have.